css/color-scheme.css

```diff
@@ -379,6 +336,22 @@ body {
     align-items: center;
 }
 
+/* typography */
+
+.typography-base {
+    text-align: center;
+    flex-direction: column;
+    --gap: 0;
```

Comment:
i didn't understand what is this?

Reply:
I used an utility class for flex, but wanted to add some custom properties to that, so defined those, also the gap variable is also defined in the utility class with a fallback of 1rem.